<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QDialog" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>187</x>
     <y>370</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <widget class="QSpinBox" name="quantity_hash_table">
    <property name="minimumSize">
     <size>
      <width>77</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>77</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="AVL_TREE_ENTRANCE">
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Введите размер хэш-таблицы&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>371</width>
     <height>161</height>
    </rect>
   </property>
   <property name="plainText">
    <string>РАЗРАБОТКА ИНФОРМАЦИОННОЙ СИСТЕМЫ ДЛЯ РАБОТЫ СО СПРАВОЧНИКОМ ПРЕДМЕТНОЙ ОБЛАСТИ «УЧЁТ УСПЕВАЕМОСТИ СТУДЕНТА»

Разработал: студент второго курса
Б9122-09.03.04
Талыбов Абдул Ашраф оглы</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
